homeassistant/components/alexa/handlers.py

```diff
@@ -149,6 +154,11 @@
     service = SERVICE_TURN_OFF
     if entity.domain == cover.DOMAIN:
         service = cover.SERVICE_CLOSE_COVER
+    elif domain == vacuum.DOMAIN:
+        supported = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
+        power_features = vacuum.SUPPORT_TURN_ON | vacuum.SUPPORT_TURN_OFF
+        if not supported & power_features:
+            service = vacuum.SERVICE_RETURN_TO_BASE
```

Comment:
I have not reviewed the integrations to see if this is true, but there is an assumption here that if the vacuum device supports `SERVICE_RETURN_TO_BASE` without checking for the `SUPPORT_RETURN_HOME` bit in the handler or the discovery. 

This assumption would mean that any vacuum integration that does not `(SUPPORT_TURN_ON and SUPPORT_TURN_OFF)` but `(SUPPORT_START and SUPPORT_STOP)` would also have a dock/base?

Further more, vacuum devices that `SUPPORT_TURN_OFF` but also ``SUPPORT_RETURN_HOME` could benefit from handling the _"Turn off"_ utterance the same way. But maybe that's up to the integration to handle that in the service call.